### PR TITLE
feat: set real data to BaseCountOfFindings widget 

### PR DESCRIPTION
**Pull request by yuda110**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [ ] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [ ] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
S...et...Real...Dat...a....
스크린샷 2023-06-13 오후 3 02 36

### Things to Talk About
- reference should be applied. (ex. project_s2352fsdf -> 'Belkin Snow')

### CODE REVIEW

File: apps/web/src/services/dashboards/widgets/_base/base-count-of-findings/BaseCountOfFindingsWidget.vue [hunk 1]
```diff
@@ -80,83 +81,95 @@ const props = defineProps<WidgetProps>();
...
+    groupByKey: computed<string>(() => {
+        // NOTE: When a dot(".") is included in the groupBy field, the API return value will only include the portion of the field that appears after the dot.
+        // ex. additional_info.service -> service
+        if (!state.groupBy) return undefined;
+        const dotIndex = state.groupBy.indexOf('.');
+        if (dotIndex !== -1) {
+            return state.groupBy.slice(dotIndex + 1);
+        }
+        return state.groupBy;
+    }),
```
[the remark this answers is not in the document]
Comment (yuda110) [reply apps/web/src/services/dashboards/widgets/_base/base-count-of-findings/BaseCountOfFindingsWidget.vue:94]: This only occurs in **some asset widgets**.
If there is a need for more converted `groupBy` later on, I plan to add `groupByKey` to the `useWidgetState` hook.

[the remark this answers is not in the document]
Comment (WANZARGEN) [reply apps/web/src/services/dashboards/widgets/_base/base-count-of-findings/BaseCountOfFindingsWidget.vue:94]: Thanx for explanation!

File: apps/web/src/services/dashboards/widgets/_base/base-count-of-findings/BaseCountOfFindingsWidget.vue [hunk 2]
```diff
@@ -80,83 +81,95 @@ const props = defineProps<WidgetProps>();
...
-                    },
-                },
-                sort: [{ key: 'total_findings_count', desc: true }],
-            };
-        }
-        return await SpaceConnector.clientV2.inventory.cloudService.analyze({
-            query: {
-                ...apiQuery,
+                field_group: ['key'],
+                sort: [{ key: 'value', desc: false }],
```
Comment (WANZARGEN) [inline apps/web/src/services/dashboards/widgets/_base/base-count-of-findings/BaseCountOfFindingsWidget.vue:143]: description:

### full query example

```json
  "query": {
    "filter": [
      {
        "k": "ref_cloud_service_type.labels",
        "v": "Compliance",
        "o": "eq"
      },
      {
          "k": "key",
          "v": ["pass_finding_count", "fail_finding_count"],
          "o": "in"
      }
    ],
    "group_by": ["key","unit", "region_code"],
    "granularity": "MONTHLY",
    "start": "2023-06",
    "end": "2023-06",
    "fields": {
      "value": {
        "key": "value",
        "operator": "sum"
      }
    },
    "field_group": ["key"],
    "sort": [{ "key": "value", "desc": false }]
  }
}
```

### api spec

CloudServiceStatsAnalyzeQuery [#](https://cloudforet-io.github.io/api-doc/inventory/v1/CloudServiceStats/#cloudservicestatsanalyzequery)
query (TimeSeriesAnalyzeQuery) Required
domain_id (string) Required

TimeSeriesAnalyzeQuery [#](https://cloudforet-io.github.io/api-doc/core/v1/Query/#timeseriesanalyzequery)
granularity (Granularity) Required
start (string) Required
end (string) Required
group_by (string) Repeated Required
field_group (string) Repeated Required
filter (Filter) Repeated Required
filter_or (Filter) Repeated Required
page (Page) Required
sort (SortKey) Repeated Required
fields (Struct) Required
select (Struct) Required
keyword (string) Required

CloudServiceStatInfo [#](https://cloudforet-io.github.io/api-doc/inventory/v1/CloudServiceStats/#cloudservicestatinfo)
query_set_id (string) Required
key (string) Required
value (float) Required
unit (string) Required
provider (string) Required
cloud_service_group (string) Required
cloud_service_type (string) Required
region_code (string) Required
account (string) Required
additional_info (Struct) Required
project_id (string) Required
domain_id (string) Required
created_at (string) Required

**Review by WANZARGEN — APPROVED** [review]
LGTM

**Review by sulmoJ — APPROVED** [review]
I tried to understand the API filters for chart data but failed. 😢 
I only checked the code conventions.

**Pull request merged**